### ISSUE TITLE
How many training epochs?

### ISSUE THREAD

**Issue by zjysteven** [YisenWang/MART#3]
Hi,

Thanks for making the code public and it is really concise and readable. I'm just wondering how many epochs you train the models since I didn't find it in the paper. Based on the learning rate schedule (decay at 75 and 90 epoch), I guess you are following the TRADES and therefore train for 100 epochs, right? And are ResNets and WideResNets share the same number of training epochs?

Thanks

**Comment by YisenWang** [comment]
Hi Jingyang, Yes. The total epoch is 100.

**Comment by zjysteven** [comment]
Got it. Thanks again for releasing the code. It is a nice work!

**Comment by MTandHJ** [comment]
> Hi Jingyang, Yes. The total epoch is 100.

But the default epochs of codes are 120 and 90 for training resnet and wide resnet ?
